OneCommunityGlobal/HighestGoodNetworkApp
# Shihao - Lint Fix Login

# Description
Lint Fix `src/component/Login/**`

## Main changes explained:
- Change the code style according to ESLint and Prettier

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. go to Login page
4. check the app works as normal
5. set up ESLint and Prettier in VSCode
6. check all files in `src/component/Login/**` doesn't have errors

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/62367016/7c209394-a536-4ac6-bda4-e5df3d6b2fad
